tcms/testruns/views.py

```diff
@@ -555,7 +560,9 @@ def get(request, run_id, template_name='run/get.html'):
     # Get tag list of testcases
     # 7. get tags
     # Get the list of testcases belong to the run
-    test_cases = [test_case_run.case_id for test_case_run in test_case_runs]
```

Comment:
FYI: this list comprehension looks good as it is so for such cases (1 line, easy to read, no lambdas) you can leave them as-is.

Reply:
I'm trying to be consistent, so I am removing the list comprehension everywhere. Also, I am not really comfortable with comments lying around in the code. We can think of adding a `pylintrc` file that overrides can some basic pylint settings. This way we can reduce the flags that are in the code. But for situations like these, the options are either a comment or to remove the comprehension.